task_tracking.md

```diff
@@ -0,0 +1,80 @@
+
+  - For continuous delivery, we use Kanban. Kanban emphasizes visualizing tasks and feedback, real-time communicating and maximizing efficiency. There is no time constraint in Kanban, instead, the approach encourages to get things done in the shortest time within a set sprint, this allows the team to respond to changes faster, hence, optimizing workflow. If there is an issue, it needs to be raised in the task card or via Discord channel, so the other members can participate in resolving the problem and quickly move on to the next task.
+
+  - Tasks are broken down from biggest objectives (initiatives/requirements) to the smallest details (user stories), so that they are visible, manageable and achievable.
+
+**Part B task planning and delegation will be explain more here.....**
+
+## Kanban board
+
+We use Trello as our project management tool, please check out [Trello board](https://trello.com/b/TE5Q9ZYj/t3a2-%F0%9F%8C%B8sakura-pantry).
+
+Below is the structure of our Kanban board with columns or lanes that represent different stages of work progress:
+
+**Project requirements**
+
+There are three columns for listing the criterion of project requirements, categorized into Documentation, Development, Code and Presentation. The columns are not subject to changes, they are kept for reference purposes and will only be screenshotted once.
+
+**Backlog**
+
+These are tasks that we want to do but not ready to work on. This is the staging area, where we should sort out all specifications and design relating to the task.
+
+**Source Code**
+
+This column is used for coding tasks only. It will be useful when app features are broken down into smaller tasks.
+
+**To-Do**
+
+Once all specifications are sorted out, a task will be moved from **Backlog** to **To-Do**. Here we will discuss and delegate tasks to each member. Tasks in **To-Do** are prioritized and need to be done as soon as possible.
+
+**Doing**
+
+Tasks under this column are things that we are working on, a start date is set when the task gets started and an end date for when the task is finished.
+
+**Need Review**
+
+Once a member is done with a task, they must move to **Need Review** column for other member to review before merging into main branch.
+
+**Testing**
+
+This column is used for code testing related tasks.
+
+**Done**
+
+This column is used for tasks that are completed, reviewed and ready to deliver to production.
+
+## Tasks description
+
+There are three main tasks in the development process:
+
+- Stage 1: Initial Design and Planning
+- Stage 2: Coding and Testing
+- Stage 3: Documentation
+
+See the below screenshots of tasks tracking on Trello for the Stage 1:
+
+For Stage 1, we set one week as one sprint.
+
+### Sprint 1 (Day 1 to Day 7)
```

Comment:
Sorry if you already mentioned below
I think it's better to make it clear what day of the week we start our sprint. I prefer to start on Tuesday or Wednesday. I'd like to avoid starting  our sprint on Monday because of  Monday blues

Reply:
sure, i'll add in and set as Tuesday, is that ok?